CGPs/cgp-0134.md

```diff
@@ -0,0 +1,108 @@
+---
+cgp: 0134
+title: "Validators rewards - changes in epoch parameters"
+date-created: 2024-05-04
+author: '@Thylacine @Roman @Patrick @Dee>'
+status: DRAFT
+discussions-to: https://forum.celo.org/t/draft-validators-rewards-change-in-epoch-parameters/7702
+governance-proposal-id: 175
+date-executed: 
+---
+ 
+## Overview
+
+This proposal seeks support to change the epoch parameters in Celo. The reason is that the validator rewards have gone down by 56%, the annual rewards are close to $33k and it continues to further go down. 
+ 
+The situation is not sustainable. Many of the small independent validators are coming from the Baklava Stake Off organised in 2019. The hardware requirement has more than quadrupled, all the business operation costs have gone up and maintaining a secured 24/7 operation in a high inflationary environment with regulation uncertainty is a challenge for many of us.
+ 
+## Background Information
+The chain was launched with 600m Celo at genesis in April 2020. The remaining 400m were reserved for epoch rewards. These epoch rewards are currently going to voting rewards, validator rewards, community fund and carbon offset fund. 
+
+If we look at the circulating supply, we have a total of 690m Celo so that means that the chain has created new 90m Celo since genesis. This means our annual inflation rate has been less than 3.6% and current network inflation rate is 2.09%. There are about 40k new Celo minted daily. By looking at the code, there was supposed to be a linear reward of 200m over 15 years so that’s about 36.5k Celo per epoch and 13.3m Celo per year. 
+
+As per the initial token release schedule, the community fund would have received 171m Celo over the span of 30 years. There was an assumption that if there are low voting rewards then people would not lock their Celo and this could be a security risk but reality is that people have been rather insensitive to the voting rewards and very few people know the current rewards. The community fund is set to receive a total of an extra 120m Celo from the Mento reserve.
+
+## Proposed Changes
+ 
+This proposal is to revert back and freeze the reward multiplier to its previous value two years ago. The following changes will be implemented:
+- Reward multiplier will be freezed and re-adjusted to 0.78. This will stabilize validators rewards around $58.5k
```

Comment:
Thanks @rcroessmann. I will reach out to the proposer and let them know.

Reply:
As technically, it's not possible to freeze the reward multiplier to 0.78 and we can arrive to a similar result by freezing RM to 1 and change the target validator epoch payment to 0.78*75k, we can rephrase the following bits in the _Proposed Changes_:

> - Validators Celo Epoch rewards will be adjusted from the initial $75k to $58.5k (0.78*$75k) and the reward multiplier will be freezed and brought back to 1. This will stabilize validators rewards around $58.5k per year. 
> - To account for the total of 120m Celo being transferred from Mento, this proposal will significally cut down the emissions to the community fund

Reply:
I have adjusted the phrasing as suggested. thanks @Dee297